### PR TITLE
Rename module

### PR DESCRIPTION
**Pull request by manoranjith**
#### Description

Rename the module from "dst-go" to "perun-node". Import paths, references in code, config files, license headers, readme file and contributing file are renamed accordingly. 

Note: The project name is still left as DST (unchanged) in Contributing.md.

##### Category

Implementation Task

#### Testing

- CI Pipelines passes. 
- Changes in Readme.md, Contributing.md to be accepted in review.

##### Steps to run the tests

NA

#### Checklist 

- [x] Name is added to the NOTICE file, if it is not present already.
- [x] Changes are rebased onto the target branch.

### CODE REVIEW

File: blockchain/ethereum/doc.go [hunk 1]
```diff
@@ -27,12 +27,12 @@
...
 // In order to provide this isolation the exported methods in this packages
-// use only those types defined in the root package of dst-go and in std lib.
-// This restriction enables the other packages in dst-go to compile this package
+// use only those types defined in the root package of this project and in std lib.
+// This restriction enables the other packages in perun-node to compile this package
 // as plugin and load the symbols from it in runtime (using "plugin" library)
 // without importing any package from "go-perun/backend/ethereum" or
 // "go-ethereum" or this package.
-// This enables the possibility to generate a dst-go node binary that does not
+// package of this projects the possibility to generate a perun-node binary that does not
```
Comment (choeppler) [inline blockchain/ethereum/doc.go:35]: I don't understand the first change in this line.
```suggestion
// This allows to generate a perun-node binary that does not
```

File: session/doc.go [hunk 2]
```diff
@@ -14,7 +14,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package node implements a node that can run multiple sessions.
-// Each session runs an instance of state channel network client and
-// is owned by the user of the session.
-package node
+// Package session implements a session to which a user can attach its
```
Comment (choeppler) [inline session/doc.go:17]:
```suggestion
// Package session implements a session to which a user can attach his or her
```

File: session/doc.go [hunk 3]
```diff
@@ -14,7 +14,7 @@
...
+// credentials, open and use state channels.  Each session runs an instance of
```
Comment (choeppler) [inline session/doc.go:18]: The sentence is unclear. Are open and used state channels attached to the session or are state channels opened and used within a session?

File: blockchain/ethereum/doc.go (hunk 1 above)
Comment (manoranjith) [reply blockchain/ethereum/doc.go]: The first part should have been left unchanged. I have updated the comments in 8ceb38c45c445d982c169529e179557a6d5397f2.

File: session/doc.go (hunk 2 above)
Comment (manoranjith) [reply session/doc.go]: Updated in dfea382fa8caf5a8f96bda0464c5589d96e617ce.

File: session/doc.go (hunk 3 above)
Comment (manoranjith) [reply session/doc.go]: State channels are opened, used within a session. Have updated in dfea382fa8caf5a8f96bda0464c5589d96e617ce.

**Pull request merged**